File: Data Structures & Algorithms/longest-consecutive-sequence/submission-1.cpp
class Solution {
public:
    int longestConsecutive(vector<int>& nums) {
        unordered_set<int> us(nums.begin(),nums.end());
        int res=0;
        for(int n:nums){
            if(us.find(n-1)==us.end()){
                int l = 1;
                while(us.find(n+l)!=us.end()){
                    l++;
                }
                res=max(res,l);
            }
        }
        return res;
    }
};

File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-1.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        n = nums[0]
        ind = 1
        i = 1
        while i<len(nums):
            while i<len(nums) and nums[i]==n:
                i+=1
            if i==len(nums): break
            nums[ind] = nums[i]
            ind+=1
            n = nums[i]
            i+=1
        return ind